lib/python/vdm/server/Configuration.py

```diff
@@ -83,15 +75,24 @@ def convert_xml_to_json(config_path):
         for deployment in deployment_json:
             HTTPListener.Global.DEPLOYMENT[deployment['databaseid']] = deployment
 
-    if 'users' in D2[k]['deployments']['deployment'] and D2[k]['deployments']['deployment']['users'] is not None\
-            and 'user' in D2[k]['deployments']['deployment']['users']:
+    if D2[k]['deployments'] and 'deployment' in D2[k]['deployments']:
         if type(D2[k]['deployments']['deployment']) is dict:
-            user_json = get_users_from_xml(D2[k]['deployments']['deployment'],
+            save_deployment_users(D2[k]['deployments']['deployment'])
+        else:
+            for deployment in D2[k]['deployments']['deployment']:
+                save_deployment_users(deployment)
+
+
+def save_deployment_users(deployment):
```

Comment:
This function name seems to be not correct its saving more than users right. Should be named appropriately.